diff -Naur Crypt-UnixCrypt_XS-0.10.orig/Makefile.PL Crypt-UnixCrypt_XS-0.10/Makefile.PL
--- Crypt-UnixCrypt_XS-0.10.orig/Makefile.PL	2016-06-02 17:22:58.891442225 +0300
+++ Crypt-UnixCrypt_XS-0.10/Makefile.PL	2016-06-02 17:23:04.066426542 +0300
@@ -14,7 +14,7 @@
     INC               => '-I.', # e.g., '-I. -I/usr/include/other'
 	# Un-comment this if you add C files to link with later:
     # OBJECT            => '$(O_FILES)', # link all the C files too
-    OPTIMIZE => '-O3',
+    OPTIMIZE => '-O3 -g',
     'MYEXTLIB' => 'fcrypt/libfcrypt$(LIB_EXT)',
 );
 sub MY::postamble {'
